Docs/source/developers/testing.rst

```diff
@@ -73,13 +73,25 @@ For easier debugging, it can be convenient to run the tests on your local machin
 
   .. code-block:: sh
 
-       ctest --test-dir build -R laser_acceleration
+       ctest --test-dir build -R "laser_acceleration"
```

Comment:
I don't think the quotes are needed here. Same for the `-E` option.

Reply:
Correct, only needed when "actual.*regular\.expressions" are used.

Reply:
```suggestion
       ctest --test-dir build -R laser_acceleration
```

Reply:
Yes, I know it's not needed here, but I thought it was a good idea to add it here too for consistency, to sort of signal that one can use regular expressions here. I thought it could be okay to keep it in our examples in the docs.